src/app/shared/components/button/button.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss'],
+})
+export class ButtonComponent {
+  @Input() public buttonLabel = '';
+  @Input() public isDisabled = false;
+  @Input() public btnClass = '';
+  @Input() public svgPlacement: 'start' | 'end' | 'none' = 'none';
+  @Input() public borderColor?: string;
+
+  @Output() public buttonClick: EventEmitter<void> = new EventEmitter();
+
+  public getSvgPlacementMargin(): string {
```

Comment:
I think you don't need this function, just pass margin values depending on svgPlacement